### PR TITLE
set invalid time stamp after issuing eofr, do not allow consecutive e…

### PR DESCRIPTION
**Pull request by hassanctech**
…ofrs

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/client/include/com/amazonaws/kinesis/video/client/Include.h [hunk 1]
```diff
@@ -184,6 +184,7 @@ extern "C" {
 #define STATUS_CLIENT_SHUTTING_DOWN                                                 STATUS_CLIENT_BASE + 0x00000087
 #define STATUS_PUTMEDIA_LAST_PERSIST_ACK_NOT_RECEIVED                               STATUS_CLIENT_BASE + 0x00000088
 #define STATUS_NON_ALIGNED_HEAP_WITH_IN_CONTENT_STORE_ALLOCATORS                    STATUS_CLIENT_BASE + 0x00000089
+#define STATUS_TWO_CONSECUTIVE_EFOR                                                 STATUS_CLIENT_BASE + 0x0000008a
```
Comment (MushMal) [inline src/client/include/com/amazonaws/kinesis/video/client/Include.h:187]: minor: suggest STATUS_MULTIPLE_CONSECUTIVE_EOFR

Comment (hassanctech) [inline src/client/include/com/amazonaws/kinesis/video/client/Include.h:187]: needs to be EOFR instead of EFOR will fix this

**Review by MushMal — COMMENTED** [review]
The change looks good. Do you mind adding a unit test to hit this double eofr? There must be a bunch of unit tests that manually submit it. You can enhance it to submit another Eofr and expect it to fail with exact this error.

**Review by MushMal — APPROVED** [review]

**Review by MushMal — APPROVED** [review]

**Review by MushMal — APPROVED** [review]
We really need to get to the bottom of these issues on Windows at some stage. Not sure if this is an indication of a test or a fixture problem. Is there a way we could repro these locally?

**Pull request merged**